proteus/mprans/cMCorr2.pyx

```diff
@@ -1,3 +1,4 @@
+# A type of -*- python -*- file
```

Comment:
@manuel-quezada I just want you to check that there's nothing important in here before you remove it, assuming this was mistakenly added. Otherwise this is ready to go in.